src/electionguard/guardian.py

```diff
@@ -363,8 +363,10 @@ def verify_election_partial_key_backup(
         """
         backup = self._guardian_election_partial_key_backups.get(guardian_id)
         public_key = self._guardian_election_public_keys.get(guardian_id)
-        if backup is None or public_key is None:
-            return None
+        if backup is None:
+            raise ValueError(f"No backup exists for {guardian_id}")
+        if public_key is None:
+            raise ValueError(f"No public key exists for {guardian_id}")
```

Comment:
Once upon a time the electionguard repo avoided throwing exceptions to ease of transcribing to languages that don't support exceptions.  I see no reason to continue that pattern, but I will revert if anyone disagrees.

Reply:
Agreed.  Exceptions will be put back until it needs to be discussed.